posit-dev/positron
# Move the `hasRunningNotebookSession` context out of the notebook session service

Another step toward removing the notebook session service entirely (#2671).

### QA Notes

This shouldn't change any behavior, although I did fix some edge cases where the context was not set correctly.

Opening a notebook, restarting it, closing and reopening it should all work. The restart button should be deactivated when the session is exited.
